src/ucp/core/ucp_worker.c

```diff
@@ -2391,9 +2391,8 @@ static void ucp_worker_discard_uct_ep_cleanup(ucp_worker_h worker)
     uct_ep_h uct_ep;
 
     /* Destroying UCP EP in ucp_ep_disconnected() could start UCT EP discarding
-     * operations (e.g. for TMP EP in CM lane). Do cleanup of discarding
-     * functionality after trying to destroy UCP EPs in order to destroy all
-     * remaining UCP EPs here */
+     * operations. Do cleanup of discarding functionality after trying to
+     * destroy UCP EPs in order to destroy all remaining UCP EPs here */
```

Comment:
is really UCP ep meant here (in the last phrase)? Looks like should be UCT ep

Reply:
no, this is correct. here we destroying UCP EPs.
but added extra clarification to the comment